### ISSUE TITLE
Docker volume filter to delete local&dangling volumes only

### ISSUE THREAD

**Issue by h0tbird** [katosys/kato#3]
Wait for Docker `1.12.0` (docker/docker/pull/21361) and use:

```
docker volume rm $(docker volume ls -q -f driver=local -f dangling=true)
```

By doing so, the garbage collector won't delete `REX-Ray` volumes.
Only `local` volumes will be deleted.

**Comment by h0tbird** [comment]
Ops! This fix was for #4

**Comment by h0tbird** [comment]
`docker volume rm $(docker volume ls -f dangling=true | awk '/^local/ {print $2}')`
